snakedeploy/providers.py

```
@@ -54,7 +55,16 @@ def clone(self, tmpdir: str):
         """
         A local "clone" means moving files.
         """
-        copy_tree(self.source_url, tmpdir)
+    def clone(self, tmpdir: str):
```

Ruff (0.12.2)

Redefinition of unused `clone` from line 54

(F811)